conversion/conversion.go

```diff
@@ -417,12 +417,85 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+// VerifyDb checks whether the db exists and if it does, verifies if the schema is what we currently support.
+func VerifyDb(project, instance, dbName string) (dbExists bool, err error) {
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return false, fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
+	dbURI := fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName)
+	dbExists, err = CheckExistingDb(ctx, adminClient, dbURI)
+	if err != nil {
+		return dbExists, err
+	}
+	if dbExists {
+		err = ValidateDDL(ctx, adminClient, dbURI)
+	}
+	return dbExists, err
+}
+
+// CheckExistingDb checks whether the dbURI exists or not.
+func CheckExistingDb(ctx context.Context, adminClient *database.DatabaseAdminClient, dbURI string) (bool, error) {
```

Comment:
make the methods private where it is not expected to be called from outside

Reply:
These are being called in the unit tests thats why kept it public.